File: templates/python/power_system_deep_before_after.py
"""power_system_deep_before_after: 电力系统深化前后斜率对比（slope 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='power system analysis: before-after slope'):
    return make_template_figure('slope', seed=3620, title=title, domain='power system analysis', topic='before-after slope')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)

File: templates/matlab/power_system_deep_before_after.m
function fig = power_system_deep_before_after()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('slope', 3620, 'power system analysis: before-after slope', 'power system analysis', 'before-after slope');
end
